templates/kubernetes/Main.scala

```diff
@@ -1,12 +1,12 @@
 import besom.*
-import besom.api.kubernetes.apps.v1.{Deployment, DeploymentArgs}
+import besom.api.kubernetes.apps.v1.{deployment, DeploymentArgs}
 import besom.api.kubernetes.core.v1.inputs.{ContainerArgs, ContainerPortArgs, PodSpecArgs, PodTemplateSpecArgs}
 import besom.api.kubernetes.meta.v1.inputs.{LabelSelectorArgs, ObjectMetaArgs}
 import besom.api.kubernetes.apps.v1.inputs.DeploymentSpecArgs
 
 @main def main = Pulumi.run {
   val appLabels = Map("app" -> "nginx")
-  for nginxDeployment <- Deployment(
+  for nginxDeployment <- deployment(
```

Comment:
wait, what?

Reply:
beta thawa was released has old syntax as mentioned on slack, will update to 0.2 in next step